src/main/java/hudson/maven/MavenModuleSetBuild.java

```diff
@@ -924,7 +924,7 @@ protected Result doRun(final BuildListener listener) throws Exception {
                 return Result.FAILURE;
             } catch (RuntimeException e) {
                 // bug in the code.
-                e.printStackTrace(listener.error("Processing failed due to a bug in the code. Please report this to jenkinsci-users@googlegroups.com"));
+                e.printStackTrace(listener.error("Processing failed due to a bug in the code. Please report this to the issue tracker"));
```

Comment:
Or simply

```java
e.printStackTrace(listener.error("Processing failed unexpectedly"));
```

If the user is motivated enough to capture details and steps to reproduce and so on, they will figure out how to use the issue tracker. Given that we are swamped by vague and unreproducible bugs as it is I am reluctant to make it sound simpler than it is.